src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -26,54 +26,108 @@ const { iotPrefix } = settings;
  * @returns {object} with a labels array and a datasets array
  */
 export const formatChartData = (series, values) => {
-  let labels = [];
-  let datasets = [];
+  const data = [];
```

Comment:
Can we get a testcase around this function?  Specifically to cover the two cases

Reply:
Added